project_management_system/src/app/app.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { LanguageService } from './services/language.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
```

Comment:
According to best practices, it better to use onPush strategy ( changeDetection: ChangeDetectionStrategy.OnPush) for each component. That helps you to prevent performance issue with application.